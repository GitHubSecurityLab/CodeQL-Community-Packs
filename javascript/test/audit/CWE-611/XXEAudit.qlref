audit/CWE-611/XXEAudit.ql
