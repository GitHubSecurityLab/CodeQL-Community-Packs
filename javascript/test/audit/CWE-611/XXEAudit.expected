| xxe.js:6:19:6:39 | req.par ... e-xml") | XML External Entity sink |
| xxe.js:7:25:7:45 | req.par ... e-xml") | XML External Entity sink |
| xxe.js:11:20:11:40 | req.par ... e-xml") | XML External Entity sink |
| xxe.js:15:13:15:33 | req.par ... e-xml") | XML External Entity sink |
